docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc

```diff
@@ -1252,54 +1252,77 @@ generate an example key pair:
 /subsystem=elytron/key-store=httpsKS:read-aliases(recursive=true,verbose=true)
```

Comment:
Hi @theashiot , please can you see this section rendered anywhere? I am using `mvn install -DskipTests -Pdocs` to generate the files, but when I then open wildfly/docs/target/generated-docs/WildFly_Elytron_Security.html I can't see this section rendered, not sure what I am missing

Reply:
Hi @Skyllarr, i use "mvn package" in the docs directory to generate docs. And I see this section in the generated docs.

I tried "install" in the WildFly directory but I get:

"[ERROR] Failed to execute goal org.wildfly.galleon-plugins:wildfly-galleon-maven-plugin:6.2.2.Final:build-feature-pack (feature-pack-build) on project wildfly-galleon-pack: Exception while reading model for io.smallrye.config:smallrye-config-parent:pom:3.1.0. Resolved pom file /home/amehenda/.m2/repository/io/smallrye/config/smallrye-config-parent/3.1.0/smallrye-config-parent-3.1.0.pom: Failed to parse artifact POM model: Unknown attribute 'child.scm.url.inherit.append.path' for tag 'scm' (position: START_TAG seen ...rit.append.path="false" child.scm.url.inherit.append.path="false">... @59:157) -> [Help 1]"

However, since all the checks are passing here, I guess the error has to do with something local.

Reply:
yup I was able to see this section rendered after using `mvn package` in docs folder. thanks!